src/actions/backup/index.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable import/prefer-default-export */
+import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { readBinaryFile, readDir } from '@tauri-apps/api/fs';
+import { join, resolve } from '@tauri-apps/api/path';
+import { PROJECTS_ARCHIVE_PREFIX } from '../../constants/paths';
+import { getProject, getStoreValue, updateProject } from '../../store';
+
+/**
+ * By default, will upload all .ai files found in a project folder
+ * to the S3 staging bucket
+ * @param {String} projectName Project slug for directory path
+ * @param {Object} opts
+ * @param {String[]} [opts.files] Adobe Illustartor file names
+ * WITHOUT the extension (e.g. `['illo-one', 'illo-two']`)
+ */
+async function backupFilesS3(projectSlug, { files } = {}) {
+  const s3 = new S3Client({
+    region: 'us-east-1',
+    credentials: {
+      accessKeyId: import.meta.env.VITE_AWS_KEY_ID,
+      secretAccessKey: import.meta.env.VITE_AWS_SECRET_KEY,
+    },
+  });
+
+  const projectsFolderPath = await getStoreValue('projectsFolder');
+
+  // Path to project folder
+  const projectPath = await resolve(projectsFolderPath, projectSlug);
+
+  const projectFiles = await readDir(projectPath);
+
+  // Removes .DS_store and any hidden files
+  // Returns array of illustration names without extensions
+  const fileNames = projectFiles
+    .filter(({ name }) => name.substring(0, 1) !== '.')
+    .map((d) => d.name);
+
+  const bucket = import.meta.env.VITE_AWS_BACKUP_BUCKET_NAME;
```

Comment:
We can either make it a constant or keep it in the `.env`. In the latter, it gets bundled together as a static value at build time instead of "imported" from our constants file

Reply:
I went ahead and moved it to a constant